Samples/Console.csproj

```diff
@@ -15,7 +15,12 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <PlatformTarget>x86</PlatformTarget>
+    <PlatformTarget>x64</PlatformTarget>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
+    <PlatformTarget>x64</PlatformTarget>
+    <DefineConstants>MAC;LINUX</DefineConstants>

```

Comment:
i thought we were removing these?

Reply:
I had removed it from another csproj earlier. Now I removed it from this file as well